README.md

```diff
@@ -17,6 +17,7 @@ Curator: [Alex Ellis](https://www.alexellis.io) - CNCF Ambassador, [OpenFaaS](ht
 
 * [bmc-toolbox](https://github.com/bmc-toolbox) - "bmc-toolbox is bunch of tools to ease BMC management"
 * [cobbler](https://github.com/cobbler/cobbler) - "Cobbler is a Linux installation server that allows for rapid setup of network installation environments"
+* [confluent](https://githum.com/lenovo/confluent) - "Service for onboarding and management of baremetal server BMCs and PXE"
```

Comment:
Sorry you have a typo. Please close and raise a new PR along with a quick note on why you feel this should be included

Reply:
githum should have been GitHub